manifests/addons/values-kiali.yaml

```diff
@@ -1,13 +1,10 @@
-# Create a Kiali CR in the istio-system namespace, which is where Kiali will be installed.
 # We set up anonymous authentication as this is for demos.
-cr:
-  create: true
-  namespace: istio-system
-  spec:
-    auth:
-      strategy: anonymous
-    deployment:
-      accessible_namespaces:
-      - '**'
-      image_version: v1.22
-      ingress_enabled: false
+auth:
```

Comment:
very nit: the deployment takes 30s to become ready due to readiness probe interval as 30s. May consider dropping it down?

I don't care too much about it here, but might be a more sane default unless there is context I don't have

Reply:
I can't remember why we set this to 30s. I'm sure there was a reason :) We can change it in the future if people deem it unnecessary.

Reply:
Its not a big deal at all, I am just impatient :slightly_smiling_face: anyone kicking tires on this will have image pull times, etc going on as well so not a big deal.